modules/cli/src/main/scala/coursier/cli/bootstrap/BootstrapSpecificOptions.scala

```diff
@@ -2,63 +2,82 @@ package coursier.cli.bootstrap
 
 import caseapp.{ExtraName => Short, HelpMessage => Help, ValueDescription => Value, _}
 import coursier.install.RawAppDescriptor
+import coursier.cli.options.OptionGroup
 
 // format: off
 final case class BootstrapSpecificOptions(
+  @Group(OptionGroup.bootstrap)
   @Short("o")
     output: Option[String] = None,
+  @Group(OptionGroup.bootstrap)
+  @Hidden
   @Short("f")
     force: Boolean = false,
+  @Group(OptionGroup.bootstrap)
+  @Hidden
   @Help("Generate a standalone launcher, with all JARs included, instead of one downloading its dependencies on startup.")
   @Short("s")
     standalone: Option[Boolean] = None,
+  @Group(OptionGroup.bootstrap)
+  @Hidden
```

Comment:
Same as just above, I wouldn't hide this.

Reply:
Done